src/RageSoundPosMap.cpp

```diff
@@ -177,16 +169,22 @@ std::int64_t pos_map_queue::Search( std::int64_t iSourceFrame, bool *bApproximat
 	 * 2. After GetDataToPlay returns EOF and the sound has flushed, but before
 	 *    SoundStopped has been called.
 	 * 3. Underflow; we'll be given a larger frame number than we know about.
+	 *
+	 * last.touch() replaces last.GetDeltaTime() since it was only serving to reset
```

Comment:
Looking into this and it seems like the "lot of resources" is an extra call to Difference() and a copy/assignment. 

PeekDeltaTime essentially has the following calls:
1. Ago()
  a. Touch()
    - GetTime()
2. Difference()

GetDeltaTime() calls
1. Touch()
  a. GetTime()
2. Difference()
4. Copy

We now just have:
1. Touch()
  a. GetTime()
  
I wouldn't really consider it a "lot" of resources, but since it is  a slight optimization we can leave it. We don't need all the historical context written as comments so can just remove it.